src/Mumble.proto

```diff
@@ -213,6 +213,8 @@ message UserState {
 	optional bool priority_speaker = 18;
 	// True if the user is currently recording.
 	optional bool recording = 19;
+	// a list of temporary acces tokens to be respected when processing this request
```

Comment:
`// A list of temporary acces tokens to be respected when processing this request.`